specification/toolingservice/data-plane/1.0/toolingservice.json

```diff
@@ -0,0 +1,755 @@
+              "description": "Missing Parameters"
+            }
+          },
+          "201": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ModelValidationResult",
+              "description": "Semantic Result"
+            }
+          },
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ModelValidationResult",
+              "description": "Semantic Result"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+    },
+    "/tools/apiValidate": {
+      "post": {
+        "tags": [
+          "semantic"
+        ],
+        "operationId": "Tools_ApiValidate",
+        "parameters": [
+          {
+            "name":"repository",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Name of the repository"
+          },
+          {
+            "name":"branch",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Name of the specific branch in the repository."
+          },
+          {
+            "name":"resourceProvider",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Name of the resource provider."
+          },
+          {
+            "name":"apiVersion",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "API Version to be validated."
+          },
+          {
+            "name":"validationId",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Validation Id"
+          },
+          {
+            "name":"startDate",
+            "in": "body",
+            "type": "string",
+            "description": "Start Date"
+          },
+          {
+            "name":"endDate",
+            "in": "body",
+            "type": "string",
+            "description": "End Date"
+          }          
+        ],
+        "responses": {
+          "400": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ErrorObject",
+              "description": "Missing Parameters"
+            }
+          },
+          "201": {
+            "description": "",
+            "schema": {
+              "properties": {
+                "validationId": {
+                  "type": "string"
+                }
+              }
+            }
+          },
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ApiSpecsValidation",
+              "description": "Semantic Result"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/tools/apiValidate/result": {
+      "get": {
+        "operationId": "Tools_ApiValidateResult",
```

Comment:
description? why we we need the post to apivalidate and then a get to apivalidate/result?